tests/functional_tests/multisig.py

```diff
@@ -156,6 +200,42 @@ def create_multisig_wallets(self, M_threshold, N_total, expected_address):
         assert res.threshold == M_threshold
         assert res.total == N_total
 
+    def remake_some_multisig_wallets_by_multsig_seed(self, threshold):
+      N = len(self.wallet)
+      signers_to_remake = set()
+      num_signers_to_remake = random.randint(1, N) # Do at least one
+      while len(signers_to_remake) < num_signers_to_remake:
+        signers_to_remake.add(random.randint(0, N - 1))
```

Comment:
I changed the function to restore *some* not *all* multisig wallets in case bugs would arise when a seed-restored interacted w/ a normal wallet.